src/Workspaces/Core/Portable/Workspace/Solution/SolutionCompilationState.CompilationTracker.cs

```diff
@@ -545,8 +549,10 @@ async Task<AllSyntaxTreesParsedState> BuildAllSyntaxTreesParsedStateFromNoCompil
                         // here from a frozen state (as a frozen state always ensures we have a
                         // WithCompilationTrackerState).  As such, we can safely still preserve that we're not
                         // frozen here.
-                        var allSyntaxTreesParsedState = new AllSyntaxTreesParsedState(
-                            isFrozen: false, compilation, CompilationTrackerGeneratorInfo.Empty, staleCompilationWithGeneratedDocuments: null);
+                        var allSyntaxTreesParsedState = InProgressState.Create(
```

Comment:
Missing rename.

Reply:
this was intentional.  it's the point at which we've parsed all syntax trees :)